### PR TITLE
Vision moved to beta stage

### PR DESCRIPTION
**Pull request by vam-google**

### CODE REVIEW

File: google-cloud/pom.xml [hunk 1]
```diff
@@ -92,7 +92,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>google-cloud-vision</artifactId>
-      <version>${project.version}</version>
```
Comment (garrettjonesgoogle) [inline google-cloud/pom.xml]: Leave the overall project at alpha.

Comment (vam-google) [inline google-cloud/pom.xml]: I was making it consistent with the other beta packages in the repo. For example for logging same file has:
    <dependency>
      <groupId>${project.groupId}</groupId>
      <artifactId>google-cloud-logging</artifactId>
      <version>${beta.version}</version>
    </dependency>

Do we really want to have it alpha here for vision?

Comment (garrettjonesgoogle) [inline google-cloud/pom.xml]: My mistake, I didn't look at the context to see that this was for vision. This is correct.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**